app/services/publishers/document_virus_check.rb

```diff
@@ -7,28 +8,37 @@ class Publishers::DocumentVirusCheck
   FILE_VIRUS_STATUS_CODE = 403 # 403 Permission denied when acknowledge_abuse: false
 
   def initialize(file)
+    @api_client = GoogleApiClient.instance
+    return unless api_client.authorization
+
+    drive_service.authorization = api_client.authorization
```

Comment:
Main change: Now we set the authorization at the service scope level. Prior was a global setting.